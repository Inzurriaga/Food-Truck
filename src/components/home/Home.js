import React from "react";
import OptionsTab from '../../containers/optionsTab/OptionsTab';
import Map from "../../containers/map/Map";

export default function Home () {
    return(
        <div className="home">
            <OptionsTab /> 
            <Map />
        </div>
    )
}