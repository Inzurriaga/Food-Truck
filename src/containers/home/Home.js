import React, { Component } from "react";
import { connect } from "react-redux"
import OptionsTab from '../../containers/optionsTab/OptionsTab';
import Map from "../../containers/map/Map";
import Loading from "../../components/loading/Loading"
import TruckList from "../../containers/trucklist/TruckList";

export  class Home extends Component{
    render(){
        return(
            <div className="home">
                <OptionsTab /> 
                {
                    this.props.loading ? 
                    <Loading/> :
                    <div className="home-main">
                        <TruckList />
                        <Map />
                    </div>
                
                }
            </div>
        )
    }
}

export const mapStateToProps = (state) => ({
    loading: state.loading
})

export default connect(mapStateToProps)(Home)

