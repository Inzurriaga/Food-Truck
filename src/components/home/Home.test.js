import React from "react";
import Home from "./Home"
import { shallow } from "enzyme";

describe("Home", () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(
            <Home />
        )
    })
    it("should match snap shot", () => {
        expect(wrapper).toMatchSnapshot()
    })
})