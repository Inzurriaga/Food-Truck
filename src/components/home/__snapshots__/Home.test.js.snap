// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match snap shot 1`] = `
<div
  className="home"
>
  <OptionsTab />
  <ConnectFunction />
  <ConnectFunction />
</div>
`;
